.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+## Bug Report
+
+**Description:**
+Provide a clear and concise description of the bug you encountered.
+
+**Expected Behavior:**
+Explain what you expected to happen when performing the steps above.
+
+**Actual Behavior:**
+Describe what actually happened instead of the expected behavior.
+
+**Steps to Reproduce:**
+List the steps to reproduce the bug. Be as specific as possible.
+
+**Environment:**
+- Operating System and Version:
+- Library Version: 
```

Comment:
Since we have the AI Verify version listed on the bug report, we can use AI Verify version to track the version for bug tracking.  Another reason for using AI Verify Version is that contributors and users are familiar with AI Verify Version since this will be what they see when pulling the release version.

Reply:
@Philip-Tan please move 'library version' to be under 'Any other relevant information' as 'e.g. version of related package dependencies installed'